packages/h/hyprpaper/package.yml

```diff
@@ -11,16 +11,18 @@ description: |
     Hyprpaper is a blazing fast wallpaper utility for Hyprland with the ability to dynamically change wallpapers through sockets. It will work on all wlroots-based compositors, though.
 builddeps  :
     - pkgconfig(cairo)
+    - pkgconfig(hyprgraphics)
     - pkgconfig(hyprlang)
     - pkgconfig(hyprutils)
     - pkgconfig(hyprwayland-scanner)
     - pkgconfig(libglvnd)
+    - pkgconfig(libjxl)
     - pkgconfig(libmagic)
     - pkgconfig(libturbojpeg)
     - pkgconfig(libwebp)
```

Comment:
It looks like it no longer links against these. Are they still needed?

Reply:
In the github page of the project these dependecies are still listed ([source](https://github.com/hyprwm/hyprpaper#dependencies)).
I checked in the Arch wiki and they still use these depencies too ([source](https://archlinux.org/packages/extra/x86_64/hyprpaper/)).

I tested to build without libwebp and cmake shows an error message which says libwebp is missing.